### PR TITLE
[WIP] Fix/minimal consensus matt

### PR DESCRIPTION
**Pull request by blazejkrzak**
**What type of PR is this?**

> Uncomment one line below and remove others.
>
> Bug fix
> Feature
> Documentation
> Other

**What does this PR do? Why is it needed?**

**Which issues(s) does this PR fix?**

Fixes #

**Other notes for review**

### CODE REVIEW

File: beacon-chain/rpc/beacon/l15-config.yaml [hunk 1]
```diff
@@ -0,0 +1,132 @@
+# Mainnet preset
```
Comment (atif-konasl) [inline beacon-chain/rpc/beacon/l15-config.yaml:1]: I think we can follow prysm's codebase. This type configuration file should be in `shared/params` module. This module is dedicated for configuration.

**Comment by atif-konasl** [comment]
cc @frozeman @meta-bot 

In your implementation, there are several issues i have found -

1. Lets say any vanguard node is running and for example, it's current epoch is 10000. In that time any orchestrator node subscribe from zero epoch number. 
2. According to your implementation, streaming api starts serving previous epochs from 0 to 10000.
3. To server these 1000 epochs, it should take minimum 10 mins. Because i have checked this time.
4. In that 10 mins, vanguard node will be advanced (10 / 3.2 ) 4 epochs ahead. So at the end of serving 10000 epochs, your vanguard node will be at 10000 ~ 10010 epoch.

I am bit confused about your implementation that how does your implementation handle this scenario?
There are other coding issues i have found. I will add more comment in this PR.

**Comment by atif-konasl** [comment]
Do we need this PR or will i close it?

**Comment by blazejkrzak** [comment]
This is deprecated and abandoned

**Pull request closed without merging**